packages/ListBox/src/helpers/invokeOnChange.js

```diff
@@ -31,18 +31,14 @@ const invokeOnChange = (onChangeCallback = () => {}, eventType = "", event) => (
   }
 
   if (state.isMulti) {
-    onChangeCallback.apply(null, [
-      ...getSelectedOptionsMulti(state),
-      { dispatch, actionTypes: sanitizeActionTypes(), eventType, ...event },
-    ]);
+    const [selected, options, current] = getSelectedOptionsMulti(state);
```

Comment:
this doesn't affect anything but it is better to remove the apply which doesn't do anything and creates confusion.